presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -208,7 +208,7 @@ public void testArrayToArrayConcat()
         assertInvalidFunction("ARRAY [ARRAY[1]] || ARRAY[ARRAY[true], ARRAY[false]]", FUNCTION_NOT_FOUND);
 
         // This query is ambiguous. The result can be [[1], NULL] or [[1], [NULL]] depending on interpretation
-        assertFunction("ARRAY [ARRAY [1]] || ARRAY [NULL]", new ArrayType(new ArrayType(INTEGER)), asList(ImmutableList.of(1), null));
+        // assertFunction("ARRAY [ARRAY [1]] || ARRAY [NULL]", new ArrayType(new ArrayType(INTEGER)), asList(ImmutableList.of(1), null));
```

Comment:
Hmm... I didn't realize we had a variant for `concat(array(E), E)`. That seems like a big mistake. It should have been named differently.